Patch by Remi Collet <rpms@famillecollet.com> for phpMyAdmin >= 3.3.0, which adds the possibility to put
config.inc.php into a FHS conform path (e.g. /etc/phpMyAdmin/) without the use of a symlink. More details
can be found at: https://sourceforge.net/tracker/?func=detail&aid=2965613&group_id=23067&atid=377411

--- phpMyAdmin-3.3.3-all-languages/libraries/common.inc.php		2010-05-10 18:32:36.000000000 +0200
+++ phpMyAdmin-3.3.3-all-languages/libraries/common.inc.php.vendor	2010-06-26 21:50:25.000000000 +0200
@@ -536,7 +536,7 @@
  * force reading of config file, because we removed sensitive values
  * in the previous iteration
  */
-$_SESSION['PMA_Config'] = new PMA_Config('./config.inc.php');
+$_SESSION['PMA_Config'] = new PMA_Config(CONFIG_FILE);
 
 if (!defined('PMA_MINIMUM_COMMON')) {
     $_SESSION['PMA_Config']->checkPmaAbsoluteUri();
@@ -586,7 +586,7 @@
 if ($_SESSION['PMA_Config']->error_config_file) {
     $error = $strConfigFileError
         . '<br /><br />'
-        . ($_SESSION['PMA_Config']->getSource() == './config.inc.php' ?
+        . ($_SESSION['PMA_Config']->getSource() == CONFIG_FILE ?
         '<a href="show_config_errors.php"'
         .' target="_blank">' . $_SESSION['PMA_Config']->getSource() . '</a>'
         :
--- phpMyAdmin-3.3.3-all-languages/libraries/vendor_config.php		2010-05-10 18:32:36.000000000 +0200
+++ phpMyAdmin-3.3.3-all-languages/libraries/vendor_config.php.vendor	2010-06-26 21:51:06.000000000 +0200
@@ -11,6 +11,11 @@
  * @package phpMyAdmin
  */
 
+ /**
+  * Path to Configuration file
+  */
+define('CONFIG_FILE', './config.inc.php');
+
 /**
  * Path to changelog file, can be gzip compressed. Useful when you want to
  * have documentation somewhere else, eg. /usr/share/doc.
